### ISSUE TITLE
accuracy calculation 

### ISSUE THREAD

**Issue by ZlaaM** [yinguobing/cnn-facial-landmark#38]
Hey.... 
I want to calculate the accuracy at each step 
eval_metric_ops = {
    "accuracy": tf.metrics.accuracy(
        labels=label_tensor, predictions=logits)
}
return tf.estimator.EstimatorSpec(
    mode=mode, loss=loss, eval_metric_ops=eval_metric_ops)

is this right ??
thanks alot

**Comment by yinguobing** [comment]
Looks good to me. Please give it a try.

**Comment by yinguobing** [comment]
This is a regression task and I don't think 'accuracy' is the right way to evaluate the performance. Mean Squared Error is still preferred.
